bot/service/thorchain_network_service.py

```diff
@@ -73,16 +73,22 @@ def get_random_seed_node_endpoint() -> str:
         {"TESTNET": "https://testnet-seed.thorchain.info", "CHAOSNET": "https://chaosnet-seed.thorchain.info"}[
             NETWORK_TYPE]
 
-    available_node_ips = requests.get(seeding_node_url).json()
+    available_node_ips = requests.get(url=seeding_node_url, timeout=10).json()
 
-    return random.choice(available_node_ips)
+    response = {}
+    for i in range(0, len(available_node_ips) * 2):
+        ip = random.choice(available_node_ips)
+        response = requests.get(url='http://' + ip +':8080/v1/health', timeout=10)
+        if response.status_code == 200:
+            return ip
+    raise BadStatusException(response)
```

Comment:
```suggestion
    random.shuffle(available_node_ips)
    response = None
    for node_ip in available_node_ips:
        response = requests.get(url='http://' + node_ip + ':8080/v1/health', timeout=10)
        if response.status_code == 200:
            return node_ip
    raise BadStatusException(response)
```

How about shuffling the ips and iterating for each? It will give us pseudorandom choice and we will always check every node. Also declaring "empty" var as `None` seems to be more natural than empty dictonary

Reply:
Thanks much better